### ISSUE TITLE
Add action shortcut mapping for `KEYCODE_DPAD_CENTER`

### ISSUE THREAD

**Issue by dzx-dzx** [Genymobile/scrcpy#5680]
- [x] I have checked that a similar [feature request](https://github.com/Genymobile/scrcpy/issues?q=is%3Aopen+is%3Aissue+label%3A%22feature+request%22) does not already exist.

**Is your feature request related to a problem? Please describe.**
I'm trying to control my TV with scrcpy. The TV do respond to `HOME`(middle mouse) and `BACK`(right mouse) action, as well as navigation(Arrow keys). But its UI and native apps do not respond to either click or `Enter` (though installed third-party apps like kodi does) when I want to press the button in focus. After some testing, I find that manually running `adb shell input keyevent KEYCODE_DPAD_CENTER` does the job. So...

**Describe the solution you'd like**
Mapping left click or `Enter` to such keyevent

**Describe alternatives you've considered**
Implementing #4569 can do that?

**Additional context**

**Comment by rom1v** [comment]
> But its UI and native apps do not respond to either click or Enter (though installed third-party apps like kodi does) when I want to press the button in focus

It does not respond to clicks? :open_mouth: 

What about <kbd>SPACE</kbd>? and (out of curiosity) `AKEYCODE_NUMPAD_ENTER`?

**Comment by dzx-dzx** [comment]
@rom1v I do find it very strange, because it does seem to respond to a physical mouse's clicks. I will try what you suggested later.

**Comment by rom1v** [comment]
OK so it works in UHID mode? (`scrcpy -M`)

**Comment by dzx-dzx** [comment]
No...which is even more confusing to me. Upon reading [the description](https://github.com/Genymobile/scrcpy/blob/master/doc/mouse.md#uhid) about I did expect it to function, but it didn't.

**Comment by dzx-dzx** [comment]
> What about SPACE?

Oh, SPACE does work...partially. It(which maps to `KEYCODE_SPACE`?) only works on the main screen.

**Comment by dzx-dzx** [comment]
There's some personal information that I'd prefer not to give away, so can I contact you through email or something?
